---
title: ' Metabarcoding'
author: "Reed Clark Benkendorf"
date: "March 3^rd^, '23"
output:
  ioslides_presentation: default
  slidy_presentation: default
subtitle: Thesis Defense
---

<style>

slide.backdrop {
  background: black;
}
body {
  background-color: black;
}
h2 { 
text-align: right;
 color: #4E2A84;		
}
h3 { 
 color: #3399ff;		
}

</style>

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = FALSE)
knitr::opts_chunk$set(message  = FALSE)
knitr::opts_chunk$set(warning = FALSE)
```

```{r}
library(tidyverse)
```



## Overview

1 | background  

2 | questions  

3 | approaches  

4 | methods  

5 | results  

6 | future  

7 | conclusions  


## Team 

-  Emily J. Woodworth (pollen morphology, and microscopy)
-  Jane Ogilvie (ecological fieldwork)
-  Sophie Taddeo (supervisory geospatial statistician)
-  Paul CaraDonna (big picture(s,) and little bees)
-  Jeremie Fant (all things molecular and tied together)

*i.e.* an in-house production. 

## the world is big - 1.1

## ... *really really* big - 1.2

## funding opportunties - 1.3

*haha made you look*

## sample the planet - 1.4

<div class="columns-2">
  - Bullet 1
  - Bullet 2
  - Bullet 3
  
  ![Primary Roads](../graphics/pictures/major_roads.resized.jpg)
</div>


## from and back to functional forms - 1.5

## plant species in ecology - 1.6

<div class="columns-2">
  - mis-identification is *very* common
  - mis-identification can lead to nebulous understandings
  - mis-identification can lead to mis-management
  
  ![*Cirsium scariosum*](../graphics/pictures/Cirsium_scariosum.resized.jpg)
</div>



## insects species in ecology - 1.7

## from organisms to interactions - 1.8

<div class="columns-2">
  - Bullet 1
  - Bullet 2
  - Bullet 3
  
  ![*Solidago spathulata* & *Megachile wheeleri*, by A. Litz](../graphics/pictures/Megachile_Solidago.resized.jpg)
</div>



## pollen records - 1.9

## metabarcoding - 1.10 

## old barcodes - 1.11

## new barcodes? - 1.12

## angiosperms 353  - 1.13

## 2 - questions  

## a353 as barcodes? - 2.1

## serverless genomics? - 2.2 

## time and species rich clades - 2.3

## are a353 semi-quantitative? - 2.4

## are floral visitation and pollen records congruent? - 2.5

## 3 -  approaches 

## Flora free data - 3.1

<div class="columns-2">
  - Bullet 1
  - Bullet 2
  - Bullet 3
  
  ![FPNW 2^nd^](../graphics/pictures/fpnw2.resized.jpg)
</div>

## Flora free data; space - 3.1.1



## Flora free data; time - 3.1.2



## custom sequence databases - 3.2


## identify pollen grains - 3.3

## queen bee pollen loads - 3.4

## 4 - methods  

<div class="columns-2">
  ![](../graphics/plots/flowchart.resized.png)
  
  - <div class = "yellow"> field work  </div>
  - <div class = "orange1"> spatial  </div>
  - temporal  
  - morphologic  
  - laboratory  
  - bioinformatic  
  - post-classification  
</div>


## study system & field work - 4.1

PICTURE OF RMBL 

## pollen morphological identification 4.2

<div class="columns-2">
  - Develop grain reference library
  - Score traits
  - write keys
  - Emily Woodworth talk at ['Not Another Specimen'](https://2021.botanyconference.org/engine/search/index.php?func=detail&aid=924) ASPT @ Botany 2021
  
  ![workflow](../graphics/assorted/mircoscopy_workflow.resized.png)
</div>

## pollen reference library 4.2.1

PICTURE OF POLLEN SLIDES FORM EMILY PRESENTATION

## pollen corbiculae loads 4.2.2

<div class="columns-2">

  - Bullet 1
  - Bullet 2
  - Bullet 3
  
  ![Corbiculae Sample](../graphics/pictures/corbiculae_slide.resized.jpg)
</div>

## molecular barcoding 4.3

## spatial analysis 4.3.1

## plant species for distribution modelling 4.3.1.1.

LINEAR REGRESSION IMAGE

## species distribution modelling - 4.3.1.2

<div class="columns-2">
  - gather records from aggregator websites - BIEN
  - prune records and model
  - GLM & GAM, RF & boosting 
  - ensemble individual models
  - [talk for analytical toolkit (PBC 470)](https://github.com/sagesteppe/Analytical_Toolkit_SDM)
  
  ![a single species](../graphics/plots/besseya_predicted.resized.png)
</div>

## sdm evaluations - 4.3.1.2

- in pipeline, True skill statistics

## temporal modelling - 4.3.2


## temporal modelling subset - 4.3.2.1

SPATIAL SUBSET PICTURE

## temporal modelling distributions - 4.3.2.2

WEIBULL PICTURE ?

##  barcode references library - 4.4

## genomics work - 4.4.1

<div class="columns-2">
  - Bullet 1
  - Bullet 2
  - Bullet 3
  
  ![workflow](../graphics/assorted/molecular_metagenomic_workflow.resized.png)
</div>

## plant genomic reference dna - 4.4.2

<div class="columns-2">
  - Bullet 1
  - Bullet 2
  - Bullet 3
  
  ![Leaf Tissue from RMH](../graphics/pictures/bernie_nelson.resized.jpg)
</div>

## pollen genomics dna - 4.4.3

<div class="columns-2">
  - Bullet 1
  - Bullet 2
  - Bullet 3
  
  ![hyb-seq](../graphics/pictures/hyb-seq.resized.jpg)
</div>

## barcoding informatics - 4.4.4

IMAGE OF COMPUTER BUILDING

## metabarcoding - 4.5


## sequence database generation - 4.5.1

<div class="columns-2">
  - Kew Tree of Life ~ ### taxa
  - US ~ xx TAXA
  
  ![database]()
</div>

## sequence assignment - 4.5.2


## semi-quantitative evidence 4.5.3



## 5 | results  

## field work

<div class="columns-2">
  - observations of Queen Bumble bees!
  - of plant unqiue plant species involved
  - corbiculae loads
  
  ![corbiculae](../graphics/plots/siteMaps.png){height=92%, width=92%}
</div>

## species distribution modelling - 5.2

```{r}
dat <- read.csv('../data/evaluation_table.csv')[,2:3]
data.frame(Metric  = dat[1:5,1],
                   Value   = dat[1:5,2],
                   Metric1 = dat[6:10,1],
                   Value1  = dat[6:10,2]
                   ) %>% 
  knitr::kable(caption = "Logistic regression assessing accuracy of SDMs",
    col.names = c('Metric', 'Value', 'Metric', 'Value'),
    booktabs = T)
```


he 554 vascular plants with biotic pollination syndromes, the 493 ML ensembles accurately predicted the presence of 362 (65.3%), incorrectly predicted the presence of 64 (11.6%), incorrectly predicted 34 true presences (6.1%) as being absent, and correctly predicted the true absence of 33 (6.0%). 

The balanced accuracy of the ensembled models is 0.627 (Sensitivity = 0.340, Specificity 0.914). Of the 554 vascular plants with biotic pollination syndromes, the 475 LM ensembles accurately predicted the presence of 286 (51.6%), incorrectly predicted the presence of 41 (14.3%), incorrectly predicted 93 true presences (16.8%) as being absent, and correctly predicted the true absence of 55 (9.9%). The balanced accuracy of the ensembled models is 0.664 (Sensitivity = 0.573, Specificity 0.754). Of the 554 vascular plants with biotic pollination syndromes in the flora 13 (2.3%) were in the Orchid family and 41 (7.4%) are non-natives, both of which are restricted from the database, and can only reduce the number of true predicted presences by roughly 10%. 


## sdm evaluations - 5.3


|           |     ml     |    lm      |
|:---------:|:----------:|:----------:|
| ensembles |      493   |      473   |
|   true +  |      362   |      286   | 
|   true -  |       33   |       55   |
|  false +  |       64   |       41   |
|  false -  |       34   |       93   |

- Plot Level, 117 species total (109 eligible for modelling...)
  - ML: 105 (89.7% (96.3%))  
  - LM: 102 (87.2% (93.5%))

## coarse phenological modelling - 5.4

<div class="columns-2">
  - strong agreement between first and peak flower periods
  - good agreement between last flower date 
  - *no* agreement with duration! - species do not 'line up'
  
  ![flower dates](../graphics/plots/phenology_estimates_10_90.png){height=92%, width=92%}
</div>

## metabarcoding - 5.5

## sequence database generation - 5.6



## sequence assignment - 5.7

## semi-quantitative evidence - 5.8


## 6 | Discussion

## 7 | future  

## new data sets? | 7.1

- artificial pollen loads
- Gunnison Sage-Grouse scat

## novel approaches | 7.2

- searching for variable loci
- flanking regions

## 8 | conclusions  

promising

## acknowledgements

two super fantastic technicians for the field seasons I worked during school, made my life SO easy   
find them funding and recruit them, and then give me a finders fee.    

- Dani Yashinowitz B.S. (Yellowstone National Park, botanist & crew lead, Whitebark Pine Surveys (!!!))
- Hannah Lovell B.S. (Telluride Mountain Resort, and ISO work)

## acknowledgments

Employment: Yingying Xie, Josh Scholll, Sam Isham, Kelly McMillen, Kay Hajek, Linda Vance, Cassandra Owen, Ken Holsinger

Project: Nyree Zerega, Pat Herendeen, Hilary Noble, Zoe Diaz-Martinez, Angela McDonnell, Elena Loke, Ian Breckheimer, Ben Legler, Ernie Nelson, Charles (Rick) Williams, D. Knoke, L. Brummer, J. Boyd, C. Davidson, I. Gilman, M. Kirkpatrick, S. McCauley, J. Smith, K. Taylor, & C. Williams. David Giblin, Mare Nazaire, Sarah Burnett, Lauren Price, T.C.H. Cole, Eliot Gardner. 


